micro_dl/inference/readme.md

```diff
@@ -1,11 +1,13 @@
-## Inference
+# Inference
 
 The main command for inference is:
+
 ```buildoutcfg
```

Comment:
The behavior of inference CLI needs to be changed, I get following error when trying to run inference.  
```
(lit-monai) [shalin.mehta@gpu-a-2 microDL]$ python micro_dl/cli/torch_inference_script.py --config /hpc/projects/CompMicro/projects/virtual_staining/models/phase2nuclei018/test/inference_config_processed_hek_no_perturb_512_512.yml 
Traceback (most recent call last):
  File "/hpc/mydata/shalin.mehta/code/microDL/micro_dl/cli/torch_inference_script.py", line 201, in <module>
    main(args.config, args.gpu, args.gpu_mem_frac)
  File "/hpc/mydata/shalin.mehta/code/microDL/micro_dl/cli/torch_inference_script.py", line 190, in main
    torch_predictor = torch_inference_utils.TorchPredictor(
  File "/hpc/mydata/shalin.mehta/code/microDL/micro_dl/inference/inference.py", line 59, in __init__
    self.network_z_depth = self.network_config["in_stack_depth"]
KeyError: 'in_stack_depth'
```

I think we are now linking `network_z_depth` and `in_stack_depth` in train.py

Reply:
Let's switch to lightning CLI's `predict` subcommand after we merge this PR.